deploy/chart/templates/plugin.yaml

```diff
@@ -24,6 +28,15 @@ spec:
                 operator: NotIn
                 values:
                 - virtual-kubelet
+{{- if $nodepool.affinity }}
+            - matchExpressions:
+              - key: {{ $nodepool.affinity.key }}
+                operator: {{ $nodepool.affinity.operator }}
+                values:
+                {{- range $nodepool.affinity.values }}
+                - {{ . | quote }}
+                {{- end -}}
```

Comment:
```
            nodeSelectorTerms: {{- toYaml $nodepool.nodeSelectorTerms | nindent 12 }}
```
Node pool selector should goes into the same `matchExpressions` as virtual-kubelet, so that they can be ANDed.
I think we can just take the hole nodeSelectorTerms from values. And move the `virtual-kubelet` things to default node pool.